src/index.html

```diff
@@ -1,17 +1,172 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width,
+                user-scalable=no,
+                initial-scale=1.0,
+                maximum-scale=1.0,
+                minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      onsubmit="onSubmit(submit)"
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="field">
+        <legend>
+          Personal information
+        </legend>
+        <label class="field__item" for="Surname">
+          Surname:
+          <input
+            id="Surname"
+            type="text"
+            name="Surname"
+            required
+          >
+        </label>
+        <label class="field__item" for="Name">
+          Name:
+          <input
+            id="Name"
+            type="text"
+            name="Name"
+            minlength="2"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field__item" for="Old">
+          How old are You?
+          <input
+            id="Old"
+            type="number"
+            name="Old"
+            value="12"
+            max="100"
+            class="numb"
+          >
+        </label>
+        <label class="field__item" for="date">
+          Full date of birth:
+          <input
+            id="date"
+            type="date"
+            name="Birth"
+            min="1"
+          >
+        </label>
+        <label class="field__item" for="accept">
+          I accept the term of the agreement
+          <input id="accept" type="checkbox" name="accept">
```

Comment:
Если у тега есть 3 и больше атрибутов - их надо записывать с новой строки
```suggestion
          <input
            id="accept"
            type="checkbox"
            name="accept"
          >
```